Telecomm/telecommServer.py

```diff
@@ -32,6 +32,25 @@
 
 #TODO
 #Document input/output values
+#Instructions for usage:
+#   1) In the registry, create the directory /Servers/Telecomm
+#   2) Create a key named 'smsUsers' with value
+#       [("USERNAME1","TENDIGITPHONENUMBER@txt.att.net"),...
+#       ,("USERNAMEM","TENDIGITPHONENUMBER@vtext.com")]
+#      Note that the domain name depends on the user's
+#      mobile phone provider. "USERNAME" must be ALL caps.
+#   3) Create a key named 'domain' with a value that
+#      matches the domain name of the email address
+#      from which you wish to send messages.
+#      e.g. domain = "physics.someschool.edu"
+#   4) Create a key named 'smtpServer' whose value is
+#      the smtp server you wish to use.
+#   5) (Optional) Create a key named 'password'
```

Comment:
I'm inclined to agree with @DanielSank -- storing a password in the registry isn't ideal.